record-and-playback/core/lib/recordandplayback/generators/video.rb

```diff
@@ -73,12 +73,8 @@ def BigBlueButton.process_webcam_videos(target_dir, temp_dir, meeting_id, output
       {
         extension: 'mp4',
         parameters: [
-          # These settings are appropriate for medium quality at any resolution
-          # Increase -threads (or remove it, to use all cpu cores) to speed up processing
-          # You can also change the preset: try 'fast' or 'faster'
-          # To change quality, adjust the -crf value. Lower numbers are higher quality.
           %w[-c:v copy
-             -c:a aac -b:a 64K
+             -c:a aac -q:a 0.5
```

Comment:
We're using the native ffmpeg aac encoder here, and the documentation currently recommends against using the vbr (quality setting) mode: https://trac.ffmpeg.org/wiki/Encode/AAC#NativeFFmpegAACEncoder

I'd suggest sticking with setting a bitrate for the time being.

Reply:
Yes, I saw that too, but it looks pretty old information. So I tested myself on a few recordings by compressing same file with ```b:a 64``` and with ```q:a 0.5```. The average correlation coefficient between b:a and q:a is about 99,99%, so they are essentially the same. In a couple of cases, the q:a version had higher correlation with the original file (recording.ogg) than the b:a.
Anyways, let me know if you want me to revert to b:a.